docs/geedocs/5.2.5/answer/1089795.html

```diff
@@ -56,6 +56,13 @@ <h3>To define map types in GEE:</h3>
 var labels_layer = new GFusionTileLayer(geeServerDefs.serverUrl, geeServerDefs.layers[2]);</code></p>
 <p>You can access the server definitions and other information about map and imagery layers directly from a web browser by using a URL in this format: <code>http://yourserver.org/virtual-server-name/query?request=Json&amp;vars=geeServerDefs</code>. Responses will look like the image below:</p>
 <p><img alt="GEE server definitions" src="../art/fusion/projects/2972270_server_definitions.jpg"/></p>
+<p>This data can also be returned by using the optional <code>v</code> parameter.o</p>
+<ul>
+<li><code>v=1</code> returns the data in the format shown in the image above.</li>
+<li><code>v=2</code> returns the data in the JSON format.</li>
+<li> A query without the <code>v</code> parameter acts as <code>v=1</code>.</li>
+</ul>
+<p>A example of a query using the <code>v</code> parameter: <code>http://yourserver.org/virtual-server-name/query?request=Json&amp;v=2</code>
```

Comment:
the url would need to have the published database entry in it. See my previous comment in the release notes about the url.

Reply:
Yeah, "virtual-server-name" should be changed to "publish-point".